### PR TITLE
Clarifies wording

### PR DESCRIPTION
**Pull request by mburger89**
I have refactored this a bit to clarify an issue that we received and I have added wording about other file formats. Let me know what you think.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 [MBK keycap profile](https://fkcaps.com/keycaps/mbk) has been designed by **mburger** and is being manufactured by [FKcaps](https://fkcaps.com). They are low profile keycaps, meant to fit Kailh Choc switches.
 
-> These models are provided for ***rendering purpose only*** and are not licensed for any commercial use.
+> These models are provided for ***rendering purpose only*** and are not licensed for any other commercial use than renderings.
```
Comment (kartsims) [inline README.md]: what do you think of 
> These models are provided for ***rendering purpose only*** and are not licensed for any commercial or manufacturing use (rendering for a commercial product is allowed).

File: README.md [hunk 2]
```diff
@@ -2,7 +2,8 @@
...
+> If you require other formats you may convert them to suite your needs in other software like mesh mixer but these convert files may only only be for personal use and may not be shared.
```
Comment (kartsims) [inline README.md]: good point, one small thing to fix please:

"only only"

File: README.md (hunk 1 above)
Comment (mburger89) [reply README.md]: Updated wording to better capture  what you mentioned. But also trying to keep it not confusing.

File: README.md (hunk 2 above)
Comment (mburger89) [reply README.md]: fixed

**Pull request merged**